charts/keycloak/values.yaml

```
@@ -166,6 +166,14 @@ readinessProbe: |
   initialDelaySeconds: 30
   timeoutSeconds: 1
 
+# Startup probe configuration
+startupProbe: |
+  httpGet:
+    path: /auth/realms/master
```

A startup probe complements the liveness probe. The liveness probe only kicks in once the startup probe has succeeded. It should, thus, use the same path as the liveness probe.

```suggestion
    path: /auth/
```

Along with that, the `initialDelaySeconds` of the liveness probe should be reduced. There's no point in having a huge delay anymore. I guess we can set it to 0.